### PR TITLE
data update

### PR DESCRIPTION
**Pull request by wingechr**
@markushal not sure if all the data came through correctly. I see some changes for DRI, but also some deletions for ATR (and in flh.csv a lot of expected deletions for PV-TRACK)

### CODE REVIEW

File: ptxboa/data/2030_high.csv [hunk 1]
```diff
@@ -9,14 +9,13 @@ key,parameter_code,process_code,flow_code,source_region_code,target_country_code
 "CALOR,,LOHC-L,,",CALOR,,LOHC-L,,,33.33,kWh H2 (l)/kg,Linde
 "CALOR,,NH3-L,,",CALOR,,NH3-L,,,5.199999999999999,kWh NH3 (l)/kg,Wikipedia with conversions
 "CAPEX,AEL-EL,,,",CAPEX,AEL-EL,,,,1210.9576551357125,USD2023/kW,
-"CAPEX,ATR,,,",CAPEX,ATR,,,,895.7895675646487,USD2023/kW,Oni et al. 2022
```
Comment (markushal) [inline ptxboa/data/2030_high.csv:12]: Looks like all parameters for ATR and REGASATR (methane reconversion with and without regasification) have been removed. I suppose this was not intentional, as we still use these processes. I will check with Roman.

File: ptxboa/data/flh.csv [hunk 2]
```diff
@@ -65,30 +62,6 @@ key,region,process_res,process_ely,process_deriv,process_flh,value,source
 "ARE,PV-FIX,SOEC-EL,NH3SYN,NH3-REC",ARE,PV-FIX,SOEC-EL,NH3SYN,NH3-REC,7008.0,öko
 "ARE,PV-FIX,SOEC-EL,NH3SYN,NH3SYN",ARE,PV-FIX,SOEC-EL,NH3SYN,NH3SYN,4809.0,PTX Atlas
 "ARE,PV-FIX,SOEC-EL,NH3SYN,SOEC-EL",ARE,PV-FIX,SOEC-EL,NH3SYN,SOEC-EL,4809.0,PTX Atlas
-"ARE,PV-TRK,AEL-EL,,H2-COMP",ARE,PV-TRK,AEL-EL,,H2-COMP,6657.599999999999,
```
Comment (markushal) [inline ptxboa/data/flh.csv:68]: removing PTR-TRK is fine. But: do we use this flh data at all? I thought this has all been replaced by profile data and optimization results.

**Review by markushal — CHANGES_REQUESTED** [review]
HI Christian, I will check with Roman regarding the ATR/REGASATR issue

**Comment by markushal** [comment]
@wingechr ATR and REGASATR had their names changed in https://github.com/agoenergy/ptx-boa/pull/526 Maybe that leads to issues when updating data in the database? Can you check that?

File: ptxboa/data/flh.csv (hunk 2 above)
Comment (wingechr) [reply ptxboa/data/flh.csv:68]: yea, its still there for backwards compatibility

**Comment by wingechr** [comment]
> @wingechr ATR and REGASATR had their names changed in #526 Maybe that leads to issues when updating data in the database? Can you check that?

so I have to rename it in the source excel file again?

**Pull request merged**